SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialWorkerConnection.cpp

```diff
@@ -9,8 +9,87 @@
 #include "Async/Async.h"
 #include "Misc/Paths.h"
 
+#if WITH_EDITOR
+#include "Editor.h"
+#include "Settings/LevelEditorPlaySettings.h"
+#endif
+
 DEFINE_LOG_CATEGORY(LogSpatialWorkerConnection);
 
+#if WITH_EDITOR
```

Comment:
The definition of the struct EditorWorkerController looks like a self-contained thing that could be moved to a separate file. Would you say it's worth doing so?

Reply:
Yep, that's valid.